# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../amanith/amanith-0.3-build.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Improted from Gentoo.

  - Rene Rebe <rene@exactcode.de>

--- amanith/plugins/fonts/build.conf
+++ amanith/plugins/fonts/build.conf
@@ -38,9 +38,9 @@
 # take into account compilation mode (debug or release)
 DEFINES -= FT_DEBUG_LEVEL_ERROR FT_DEBUG_LEVEL_TRACE
 
-INCLUDEPATH += $$(AMANITHDIR)/3rdpart/freetype2/include
+QMAKE_CXXFLAGS += `pkg-config --cflags freetype2`
 
-unix: LIBS += $$(AMANITHDIR)/lib/libfreetype2.a $$(AMANITHDIR)/lib/libzlib.a
+unix: LIBS += `pkg-config --libs freetype2` -lz
 win32: {
 
     !contains(DEFINES, WIN32_MINGW) {
--- amanith/plugins/jpeg/build.conf
+++ amanith/plugins/jpeg/build.conf
@@ -25,7 +25,7 @@
     }
 }
 
-unix: LIBS += $$(AMANITHDIR)/lib/libjpeg.a
+unix: LIBS += -ljpeg
 win32: {
 
     !contains(DEFINES, WIN32_MINGW) {
--- amanith/plugins/jpeg/gjpegimpexp.cpp
+++ amanith/plugins/jpeg/gjpegimpexp.cpp
@@ -36,7 +36,7 @@
 #include <csetjmp>
 
 G_EXTERN_C {
-	#include "../../3rdpart/libjpeg/jpeglib.h"
+	#include <jpeglib.h>
 };
 
 /****************************************************************************
--- amanith/plugins/png/build.conf
+++ amanith/plugins/png/build.conf
@@ -25,9 +25,8 @@
     }
 }
 
-INCLUDEPATH += $$(AMANITHDIR)/3rdpart/zlib
 
-unix: LIBS += $$(AMANITHDIR)/lib/libpng.a $$(AMANITHDIR)/lib/libzlib.a
+unix: LIBS += -lpng -lz
 win32: {
 
     !contains(DEFINES, WIN32_MINGW) {
--- amanith/plugins/png/gpngimpexp.cpp
+++ amanith/plugins/png/gpngimpexp.cpp
@@ -35,7 +35,7 @@
 //#include <csetjmp>
 
 G_EXTERN_C {
-	#include "../../3rdpart/libpng/png.h"
+	#include <png.h>
 };
 
 /****************************************************************************
